docs/index.md

```diff
@@ -50,6 +50,13 @@ Follow the following steps to get set up:
 
 1. Install necessary dependencies above.
 
+    ```sh
+    sudo apt-get install libprotobuf-dev
+    sudo apt-get install libprotobuf-compiler
+    sudo apt-get install libboost-all-dev
+    sudo apt-get install libopenscenegraph-dev
```

Comment:
open scene graph is only needed for mujoco and this command is listed in that section. So, let's delete this line.